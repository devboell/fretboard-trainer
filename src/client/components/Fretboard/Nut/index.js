import React from 'react'
import pt from 'prop-types'

import ViewPort from 'components/Fretboard/ViewPort'

const Nut = ({ width, offset }) =>
  <ViewPort {...{ width, offset }} />

Nut.propTypes = {
  width: pt.number.isRequired,
  offset: pt.number.isRequired,
}

export default Nut
