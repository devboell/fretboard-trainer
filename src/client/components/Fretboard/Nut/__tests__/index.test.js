import React from 'react'

import Nut from '../index'

const props = {
  width: 5,
  offset: 10,
}

it('Nut component, snapshot', () => {
  const wrapper = shallow(<Nut {...props} />)

  expect(wrapper).toMatchSnapshot()
})

