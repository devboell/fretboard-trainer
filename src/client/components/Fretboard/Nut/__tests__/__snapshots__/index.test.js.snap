// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nut component, snapshot 1`] = `
<ViewPort
  offset={10}
  width={5}
>
  <g />
</ViewPort>
`;
